### PR TITLE
Moved strings to translate from untranslated.h to specific country file

### PR DESCRIPTION
**Pull request by HThuren**
Summary of changes:
Moved strings to translate, from untranslated.h to specific country file

### CODE REVIEW

**Comment by HThuren** [comment]
HELP, I really don't know what is the issue in this conflict ?

**Comment by HThuren** [comment]
The string TR_ROTARY_ENC_MODE in concern, are changed in another PR, and I just added to get through the compilation.

**Comment by pfeerick** [comment]
When I rebase it I'll fix them up... basically just means there has been changes to the same sections of code in the upstream branch since you branched off to work on this. It'll get even worse in a minute when I merge https://github.com/EdgeTX/edgetx/pull/2186 as it looks like you've copied those strings also... but in this case, I'll probably end up deleting some of your changes... as OFF is longer in existence for those strings 😆

**Comment by HThuren** [comment]
allright, 
I'll looked into the commit made after my push locally to make the changes, bu that didn't help.
I'll look forward to your merge :-)
Thank you

**Comment by HThuren** [comment]
Have fixed #2186, moved four defines back to untranslated. But still some trouble with TR_ROTARY_ENCODER

**Comment by HThuren** [comment]
wow, I got through

**Comment by HThuren** [comment]
Wait, before merge. I have to check first, if changes are still there

**Comment by pfeerick** [comment]
git (and github) had a hissy fit because, if I'm remember it correctly, 

```
#define TR_ROTARY_ENC_MODE         "Invert Rotary"
``` 
became
```
 #define TR_ROTARY_ENC_OPT         "Normal","Inverted"
 ```
 
 in main, and at the same time the first part of the line changed to match, but the second half was different yet again, and that can't be automatically resolved as git no longer knows which is the correct change to keep, since *both* sides are different. So it's either a merge, or a rebase. Rebase is clean - you don't get all the merge commits - only actual change commits.

**Comment by HThuren** [comment]
OK; I think merge are for good now, has checked. Thanks.

File: radio/src/translations/da.h [hunk 1]
```diff
@@ -216,11 +218,39 @@
...
 #define TR_ROTENC_SWITCHES
 
 #define TR_ON_ONE_SWITCHES             "EN","En"
 
+#if defined(COLORLCD)
+  #define TR_ROTARY_ENC_OPT         "Normal","Inverted"
+#else
+  #define TR_ROTARY_ENC_OPT         "Normal","Inverted","V-I H-N","V-I H-A"
+#endif
+
```
Comment (pfeerick) [inline radio/src/translations/da.h:253]: @HThuren Did you want to do these as well? Was somehow missing... possibly got clobbered I did the rebase and resolving conflicts - I'm not sure now. Just made changes to other languages files as I just did a test build of them all and a couple were failing.

Comment (HThuren) [reply radio/src/translations/da.h:253]: Yes, very good, I also added #define TR_INVERT_ROTARY               "Invert Rotary" in da.h

**Comment by HThuren** [comment]
Also succes at myy local compile, with your changes and the last addon.
Merge, if checks are parssed - one missing rightt now, will be fine. thx

**Comment by pfeerick** [comment]
`TR_INVERT_ROTARY` is no longer used by anything... that's why it is "missing"... it was replaced with `TR_ROTARY_ENC_OPT` ;)

**Comment by HThuren** [comment]
Hi pfeerick, thank you

**Pull request merged**